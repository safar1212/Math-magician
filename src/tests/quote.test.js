import Quote from '../pages/quote';
import renderer from 'react-test-renderer';

describe('quote test', () => {
  it('quote render correctly', () => {
    const quotePage = renderer.create(<Quote />).toJSON();
    expect(quotePage).toMatchSnapshot();
  });
});
