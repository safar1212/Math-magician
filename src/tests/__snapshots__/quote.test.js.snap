// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quote test quote render correctly 1`] = `
<div
  className="QuoteDiv"
>
  <p>
    Sometimes it pays to stay in bed on Monday‚ rather than spending the rest of the week debugging Monday′s code - Dan Salomon
  </p>
</div>
`;
